src/test/java/net/minecraftforge/debug/world/MultiNoiseBiomeSourceLoadingEventTest.java

```diff
@@ -0,0 +1,259 @@
+        
+        // expected behaviour is that overworld and overworld preset json dimension have an extra end biome,
+        // nether has an extra jungle biome,
+        // and the named test dimension has an extra nether biome
+        if (Objects.equals(event.getName(), Preset.OVERWORLD.name))
+        {
+            Biome end = event.getRegistries().registryOrThrow(Registry.BIOME_REGISTRY).get(Biomes.SMALL_END_ISLANDS);
+            var parameters = event.getParameters();
+            LOGGER.info("Adding end islands biome to overworld biome source with {} biomes", parameters.size());
+            parameters.add(Pair.of(new ParameterPoint(
+                Parameter.span(-0.5F, 0.5F), // temperature
+                Parameter.span(0F, 0.5F), // humidity
+                Parameter.span(0.5F, 1F), // continentalness
+                Parameter.span(0.5F, 1F), // erosion
+                Parameter.span(0.5F, 1F), // depth
+                Parameter.span(0.5F, 1F), // weirdness
+                0L // offset
+                ), () -> end));
+        }
+        else if (Objects.equals(event.getName(), Preset.NETHER.name))
+        {
+            Biome end = event.getRegistries().registryOrThrow(Registry.BIOME_REGISTRY).get(Biomes.JUNGLE);
+            var parameters = event.getParameters();
+            LOGGER.info("Adding jungle biome to nether biome source with {} biomes", parameters.size());
+            parameters.add(Pair.of(new ParameterPoint(
+                Parameter.point(0.5F), // temperature
+                Parameter.point(0.5F), // humidity
+                Parameter.point(0.5F), // continentalness
+                Parameter.point(0.5F), // erosion
+                Parameter.point(0.5F), // depth
+                Parameter.point(0.5F), // weirdness
+                0L // offset
+                ), () -> end));
+        }
+        else if (Objects.equals(event.getName(), NAMED_TEST))
+        {
+            Biome end = event.getRegistries().registryOrThrow(Registry.BIOME_REGISTRY).get(Biomes.BASALT_DELTAS);
+            var parameters = event.getParameters();
+            LOGGER.info("Adding basalt deltas biome to test biome source with {} biomes", parameters.size());
+            parameters.add(Pair.of(new ParameterPoint(
+                Parameter.point(0.5F), // temperature
+                Parameter.point(0.5F), // humidity
+                Parameter.point(0.5F), // continentalness
+                Parameter.point(0.5F), // erosion
+                Parameter.point(0.5F), // depth
+                Parameter.point(0.5F), // weirdness
+                0L // offset
+                ), () -> end));
+        }
+    }
+    
+    private void onRegisterCommands(RegisterCommandsEvent event)
+    {
+        event.getDispatcher().register(
+            Commands.literal(MODID)
+                .then(Commands.literal("biomes")
+                    .executes(context ->{
```

Comment:
Nitpick: missing space between lambda arrow and opening brace:
```suggestion
                    .executes(context -> {
```
Alternatively, this could be a method reference to a static method to avoid this level of indentation.

Reply:
Moved into a static method in next push